homeassistant/components/group/media_player.py

```diff
@@ -208,7 +208,7 @@ def name(self) -> str:
         return self._name
 
     @property
-    def state(self) -> str | None:
+    def state(self) -> MediaPlayerState | None:
```

Comment:
Can't we reuse `self._attr_state` instead of our prop ?

Reply:
Done